axelrod/tests/unit/test_interaction_utils.py

```diff
@@ -1,12 +1,13 @@
-import tempfile
 import unittest
+
+import tempfile
+
 from collections import Counter
 
-import axelrod
+import axelrod as axl
 import axelrod.interaction_utils as iu
```

Comment:
Delete this line.  Ditto from comment on other file.

Reply:
> Ditto from comment on other file.

Can you quote the comment that you're talking about?

Reply:
import axelrod.interaction_utils as iu

It looks like it was decided to just import axelrod, and to replace iu with axelrod.interaction_utils.